patches/chrome-browser-browsing_data-chrome_browsing_data_remover_delegate.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
+index 873f56bffd068bc0fbcba6db500392e32d736b5b..f9a35337e227e252ef8034e0ccea22b8d5f98306 100644
+--- a/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
++++ b/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc
+@@ -709,6 +709,7 @@ void ChromeBrowsingDataRemoverDelegate::RemoveEmbedderData(
+           info->website_settings_info()->type(), delete_begin_, delete_end_,
+           HostContentSettingsMap::PatternSourcePredicate());
+     }
++    CLEAR_SHIELDS_SETTINGS
```

Comment:
Would patch in `ChromeBrowsingDataRemoverDelegateFactory::BuildServiceInstanceFor` to return a Brave subclass and override the RemoveEmbedderData in Brave's class to include the logic of what we want to clear more extensible?

Reply:
Done 👍